packages/oscal-react-library/src/components/OSCALParty.tsx

```diff
@@ -0,0 +1,121 @@
+import {
+  PartyOrganizationOrPerson,
+  PublicationMetadata,
+  Role,
+  PartyType,
+} from "@easydynamics/oscal-types";
+import EmailIcon from "@mui/icons-material/Email";
+import MapIcon from "@mui/icons-material/Map";
+import PhoneIcon from "@mui/icons-material/Phone";
+import PersonIcon from "@mui/icons-material/Person";
+import GroupIcon from "@mui/icons-material/Group";
+import { Card, DialogContent, DialogTitle, Grid, List, Stack, Typography } from "@mui/material";
+import React from "react";
+import {
+  MetadataAvatar,
+  MetadataInfoList,
+  MetadataInfoType,
+  OSCALMetadataCard,
+  OSCALMetadataContactTypeHeader,
+  OSCALMetadataFieldArea,
+} from "./OSCALMetadataCommon";
+
+export interface OSCALMetadataPartyProps {
+  party: PartyOrganizationOrPerson;
+  partyRolesText: Role[] | undefined;
+}
+
+export const OSCALMetadataParty: React.FC<OSCALMetadataPartyProps> = (props) => {
+  const fallbackIcon =
+    props.party?.type === PartyType.ORGANIZATION ? <GroupIcon /> : <PersonIcon />;
+
+  const avatar = (
+    <MetadataAvatar id={props.party.uuid} text={props.party.name} fallbackIcon={fallbackIcon} />
+  );
+
+  return (
+    <OSCALMetadataCard
+      title={props.party.name}
+      subheader={props.partyRolesText?.map((role: Role) => role.title).join(", ")}
```

Comment:
```suggestion
      subheader={props.partyRolesText?.map((role) => role.title).join(", ")}
```

`tsc` should be able to infer the type here